### PR TITLE
Version 1.1 of Image Metadata

### PR DESCRIPTION
**Pull request by wimbime1**
In this branch, it includes acronyms that are used in Image and Experimental Metadata. It also includes categories that has required field names and their descriptions.

### CODE REVIEW

File: image-metadata/metadata/Acronyms.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+| REM | Reflection Electron Microscope |
+| sAT | Scanning Acoustic Tomography |
+| SB_SEM | Serial Block-Face Scanning Electron Microscope |
+| SEM | Scanning Electron Microscope |
+| S_TEM | Scanning Transmission electron Microscopy |
+| SS_TEM | Serial Section Transmission Electron Microscopy |
+| STM | Scanning Tunneling Microscope |
+| TEM | Transmission Electron Microscope |
+| Xray | X-Radiation |
+| XRM | Xray Microscopy |
```
Comment (yarikoptic) [inline image-metadata/metadata/Acronyms.md:39]: @satra do you know any existing ontology/vocab which defines all those and with acronyms already?  I found some part of https://www.ebi.ac.uk/ols/ontologies/fbbi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FFBbi_00000258 but seems lacking some (e.g. MRI)

Comment (satra) [reply image-metadata/metadata/Acronyms.md:39]: the techniques and modalities ontology (https://github.com/BICCN/TMN/tree/90cf0bdd1756af1cf26d0f8d837b4a9df2a4b96d) covers  several but is not exhaustive.

Comment (satra) [reply image-metadata/metadata/Acronyms.md:39]: and this covers others: https://www.ebi.ac.uk/ols4/ontologies/edam/classes/http%253A%252F%252Fedamontology.org%252Ftopic_3361?lang=en

**Review by erikjohnson24 — APPROVED** [review]
Updated to match v1.1 and the white paper "EM and XRM Image and Experimental Metadata Standards"

**Pull request merged**